wfsim/strax_interface.py

```diff
@@ -289,6 +320,10 @@ def __init__(self, config):
                  help="Number of pmts in top array. Provided by context"),
     strax.Option('neutron_veto', default=False, track=True,
                  help="Flag for nVeto optical simulation instead of TPC"),
+    strax.Option('nv_pmt_qe_file', default='https://github.com/XENONnT/private_nt_aux_files/blob/master/sim_files/nveto_pmt_qe.json', track=True,
+                 help="path and filename to nveto PMT QE json file"),
+    strax.Option('nv_pmt_ce_factor', default=1.0, track=True,
+                 help="Collection efficiency factor for neutron Veto PMT"),
```

Comment:
These are/will be fixed values, so no chance to test nVeto under different QE/CE conditions. @zhut19 @petergaemers what's the policy here? Should the CE value simply be added to `fax_config`?

Reply:
This should go in the config. This is for plugin configuration. Simulation config should go in the config file

Reply:
Shouldn't the QE file go in `load_resource.py`? Sorry if I miss something obvious.

Reply:
The reason why I load them not in `load_resource.py` is I would like to drop optical hits in `read_optical()`. In there, `Resource` class is not available, is it right?

Reply:
I moved them to private_nt_aux_files/sim_files/fax_config_nt.json.
This implementation works for ~6,000 events MC input in dali notebook and my local environment.

Reply:
Keita, you are correct :)